_posts/2016-02-25-on-idempotency-in-distributed-rest-apis.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: post
+title: "Idempotency in REST APIs and distributed systems"
```

Comment:
Drop the 'distributed systems' part? I think Idempotency in HTTP REST APIs is enough, keeps things focused.